### PR TITLE
Bash logic fix. #149

### PR DESCRIPTION
**Pull request by jmatsushita**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ matrix:
     - node_js: '4'
       env: NO_WEB=
 script:
-  - test $NO_WEB && (npm run coverage && npm run publish-coverage) || test $SAUCE_USERNAME && npm run zuul || echo 'not running on saucelabs'
+  - test $NO_WEB && (npm run coverage && npm run publish-coverage) && ( test $SAUCE_USERNAME && npm run zuul || echo 'not running on saucelabs' )
```
Comment (mcollina) [inline .travis.yml]: That was an old-school if. I think what was missing was a parentheses after the first `&&`.

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ matrix:
     - node_js: '4'
       env: NO_WEB=
```
Comment (mcollina) [inline .travis.yml]: we can removed node v5.

File: .travis.yml (hunk 1 above)
Comment (jmatsushita) [inline .travis.yml]: Shall I let you make the semantically correct fix then? I'm not sure what was intended :)

File: .travis.yml (hunk 2 above)
Comment (jmatsushita) [inline .travis.yml]: Ok!

File: .travis.yml (hunk 1 above)
Comment (mcollina) [inline .travis.yml]: I have no time, not until next week, keep pushing here :).

**Comment by maxired** [comment]
this can probably be close

**Pull request closed without merging**